tests/components/filter/test_sensor.py

```diff
@@ -274,15 +274,15 @@ async def test_setup(hass):
             1,
             {
                 "icon": "mdi:test",
-                ATTR_DEVICE_CLASS: DEVICE_CLASS_TEMPERATURE,
-                ATTR_STATE_CLASS: STATE_CLASS_TOTAL_INCREASING,
+                ATTR_DEVICE_CLASS: SensorDeviceClass.TEMPERATURE,
+                ATTR_STATE_CLASS: SensorStateClass.TOTAL_INCREASING,
             },
         )
         await hass.async_block_till_done()
         state = hass.states.get("sensor.test")
         assert state.attributes["icon"] == "mdi:test"
-        assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_TEMPERATURE
-        assert state.attributes[ATTR_STATE_CLASS] == STATE_CLASS_TOTAL_INCREASING
+        assert state.attributes[ATTR_DEVICE_CLASS] == SensorDeviceClass.TEMPERATURE
+        assert state.attributes[ATTR_STATE_CLASS] == SensorStateClass.TOTAL_INCREASING
```

Comment:
As discussed in the other PR, please use `is` for the state class:
```suggestion
        assert state.attributes[ATTR_STATE_CLASS] is SensorStateClass.TOTAL_INCREASING
```

Reply:
Agreed. Done.